Kickstarter-iOS/Views/Controllers/PledgeTableViewController.swift

```diff
@@ -57,10 +66,24 @@ class PledgeTableViewController: UITableViewController {
 
 // MARK: - Styles
 
-private let tableViewStyle: TableViewStyle = { (tableView: UITableView) in
-  tableView
+private func tableViewStyle(_ tableView: UITableView) -> UITableView {
+  let style = tableView
+    |> \.allowsSelection .~ false
+    |> \.separatorStyle .~ UITableViewCell.SeparatorStyle.none
     |> \.contentInset .~ UIEdgeInsets(top: -35)
     |> \.sectionFooterHeight .~ 10
     |> \.sectionHeaderHeight .~ 0
-    |> \.separatorStyle .~ .none
+
+  if #available(iOS 11, *) { } else {
+    let estimatedHeight: CGFloat = 44
+
+    return style
+      |> \.contentInset .~ UIEdgeInsets(top: 30)
```

Comment:
That's a really good question.

I've decided to use `UITableView` `.grouped` style in order to not have floating headers/footers (which we thought might come handy for the section separators).

By default the `.grouped` style adds certain padding above the first row so this removes that padding. In addition to that the value of insets is different on iOS 11+ and iOS 10 (so this line is a special case - which with our deprecation plans might disappear soon). There's also a know issue on iOS 10 (when rotating to landscape) that still shows the padding - I tried couple workarounds there but without luck...so hopefully we can remove iOS 10 support before this thing gets released. Otherwise we'd have to use `.plain` style and use cells for separators.